packages/paste-website/src/pages/patterns/empty-states/index.mdx

```diff
@@ -0,0 +1,444 @@
+  ></iframe>
+</Box>
+
+### Empty state requiring a customer to wait
+
+A customer's next step might not always require an immediate call-to-action. Instead, the best path forward for them might be to wait for another process to complete and come back later. This happens most often when a customer is awaiting data through event logs, or when another person or non-Twilio service needs to approve a customer's request.
+
+If the customer might need to start another workflow, consider giving them a way back to a dashboard or the start of another flow.
+
+<Box marginBottom="space70">
+  <iframe
+    src="https://codesandbox.io/embed/empty-state-requiring-a-customer-to-wait-uyxu3?fontsize=14&hidenavigation=1&theme=dark&view=preview"
+    style="width:100%; height:500px; border:0; border-radius: 4px; overflow:hidden;"
+    title="Empty state requiring a customer to wait"
+    allow="accelerometer; ambient-light-sensor; camera; encrypted-media; geolocation; gyroscope; hid; microphone; midi; payment; usb; vr; xr-spatial-tracking"
+    sandbox="allow-forms allow-modals allow-popups allow-presentation allow-same-origin allow-scripts"
+  ></iframe>
+</Box>
+
+### Sizing & Responsiveness
+
+When an empty state card is wider than 540px (recommended), use `padding="space150"`, and align illustrations horizontally with the text content.
+
+In smaller spaces, use `padding="space100"` and stack the illustration in the card above the text content.
+
+If your empty state composition needs to accommodate different screen sizes, you can also wrap it in the Stack or Grid component.
+
+<Callout>
+  <CalloutText>
+    If you'd like to contribute guidelines for empty states in even more compact situations, like a navigation panel or
+    components like <Anchor href="/components/combobox">Combobox</Anchor>, please reach out in a{' '}
+    <Anchor href="https://github.com/twilio-labs/paste/discussions">GitHub Discussion</Anchor>!
+  </CalloutText>
+</Callout>
+
+<Box marginBottom="space70">
+  <iframe
+    src="https://codesandbox.io/embed/empty-state-sizing-responsiveness-euj5p?fontsize=14&hidenavigation=1&theme=dark&view=preview"
+    style="width:100%; height:500px; border:0; border-radius: 4px; overflow:hidden;"
+    title="Empty state requiring a customer to wait"
+    allow="accelerometer; ambient-light-sensor; camera; encrypted-media; geolocation; gyroscope; hid; microphone; midi; payment; usb; vr; xr-spatial-tracking"
+    sandbox="allow-forms allow-modals allow-popups allow-presentation allow-same-origin allow-scripts"
+  ></iframe>
+</Box>
+
+## Content examples by use case
+
+Use a heading that succinctly summarizes the situation. We recommended a maximum of 7 words.
+
+Use an illustration that's directly related to the message you're communicating.
+
+Don't use a random illustration to "fill in the blank." The more that illustrations are used throughout our products, the more they lose their emotional and visual impact when it really matters.
```

Comment:
Should these recs live elsewhere? They don't match the heading above them. Maybe the heading should be changed to include them and say "Building an Empty State" and then have another subheading above the table that says "Use Cases"

Reply:
Would changing it to this help?

> For all use cases:
> - Use a heading that succinctly summarizes the situation. We recommended a maximum of 7 words.
> - Use an illustration only if it’s directly related to the message you’re communicating. Don’t use a random illustration to “fill in the blank.” The more that illustrations are used throughout our products, the more they lose their emotional and visual impact when it really matters. Learn more about requesting a new illustration.

Eventually we'll want to rethink IA for these Patterns pages because we have composition guidance both at the top of the page and down below.